index.js

```diff
@@ -1,37 +1,33 @@
 const fs = require('fs'),
     kue = require('kue'),
     queue = kue.createQueue(),
-    BotController = require('./lib/bot_controller'),
+    CONFIG = require('./config'),
+    utils = require('./lib/utils'),
     InspectURL = require('./lib/inspect_url'),
-    ResController = require('./lib/res_controller'),
-    GameData = require('./lib/game_data'),
-    DBHandler = require('./lib/db'),
-    CONFIG = require('./config');
+    botController = new (require('./lib/bot_controller'))(),
+    resController = new (require('./lib/res_controller'))(),
+    DB = new (require('./lib/db'))(CONFIG.database_url),
+    gameData = new (require('./lib/game_data'))(CONFIG.game_files_update_interval, CONFIG.enable_game_file_updates);
```

Comment:
For the constructors that require parameters, it is extremely ugly. Maybe keep the ones with no parameters in this style?